cadc-vos/src/main/java/org/opencadc/vospace/Node.java

```diff
@@ -210,17 +209,29 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public Set<URI> getReadOnlyGroup() {
+    public Set<GroupURI> getReadOnlyGroup() {
         return readOnlyGroup;
     }
 
-    public Set<URI> getReadWriteGroup() {
+    public Set<GroupURI> getReadWriteGroup() {
         return readWriteGroup;
     }
 
     public Set<NodeProperty> getProperties() {
         return properties;
     }
+
+    @Override
+    protected boolean isDataModelClass(Class c) {
+        if (super.isDataModelClass(c)) {
```

Comment:
I think I would re-org this to check GroupURI first and if check is false, just return super.isDataModelClass(...) at the end.

for the check, hard-coding the class name will break from refactoring or a typo that the compiler won't catch.

Like this: 
```
if (GroupURI.class.equals(c)) {
  return true;
}

return super.isDataModelClass(c);
```
would be better.

Reply:
also maybe a comment like
```
// imported data model components
if (GroupURI.class ...
```

Reply:
True on both counts. Changed made.